### DISCUSSION TITLE
Https requests Query

### DISCUSSION THREAD

**Discussion by johntallon** [lowdefy/lowdefy#1140] [Questions and Help]
Hi, 

Struggling a little bit. 

Have Lowdefy authenticated with OpenId to Azure AD (Thanks for your steer on that).  Separately, I have API that is also Authenticated to the same Azure AD directory. 

I want to call that API from Lowdefy app to start some CRUD, but unsure how to use the connector and requests, as I need to send the same auth tokens to the API that Lowdefy got from Azure AD Auth. 

Do you have some info/steer or a guide I could use that will help me understand more? 

Also, how can I run Lowdefy with HTTPS rather than HTTP? 

Regards
John

**Comment by SamTolmay** [comment]
Hi @johntallon 

We have an open feature request tracking what you need: https://github.com/lowdefy/lowdefy/issues/690

This is something we want to add as part of the v4 next major version.

You would need some kind of long lived tokens that you can store as Lowdefy secrets to authenticate against the API. This would mean you need to change the API, which I usderstand is something you might not be able to do.

To run using https, first make sure you are running a production server for production. You can look at the deployment section in the docs for docs on Docker, Node.js and AWS Lambda servers. These support https. We enforce all web clients use https in Cloudflare, which we also use as DNS and CDN. You also do this using something like Nginx

**Reply by johntallon** [reply]
Thanks for the response on this @SamTolmay
